pkgs/development/compilers/mlton/default.nix

```diff
@@ -38,7 +38,7 @@ stdenv.mkDerivation rec {
 
   sourceRoot = name;
 
-  buildInputs = [ gmp ] ++ stdenv.lib.optional stdenv.isLinux patchelf;
+  buildInputs = [ gmp.out gmp.dev ] ++ stdenv.lib.optional stdenv.isLinux patchelf;
```

Comment:
Is this actually necessary? It used to be that `dev` was included automatically when a package was used as a build input.